surefire-provider/src/test/java/org/arquillian/smart/testing/surefire/provider/TestSelectionOrderingTest.java

```diff
@@ -0,0 +1,48 @@
+package org.arquillian.smart.testing.surefire.provider;
+
+import java.util.Collection;
+import java.util.List;
+import org.apache.maven.surefire.util.TestsToRun;
+import org.arquillian.smart.testing.TestSelection;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static java.util.Arrays.asList;
+
+public class TestSelectionOrderingTest {
+
+    @Mock
+    private TestsToRun testsToRun;
+
+    @Mock
+    private TestExecutionPlannerLoader testExecutionPlannerLoader;
+
+    @Test
+    public void should_order_test_selection_with_previous_strategy_configuration() {
```

Comment:
Maybe we should test what I have described in the comment related to `StrategyComparator` instead?